veeqtor/API-Trisixty
# #164209895 Get a vendor

#### What does this PR do?
- Implements get a single vendor endpoint.

#### Description of Task to be completed?
As a verified and logged in business account owner, I should be able to view all my created vendors by making a `GET` request to this route `/vendor/<vendor_id>`.

#### How should this be manually tested?
- Fetch the branch with `git fetch origin ft-view-a-vendor-164209895`
- Checkout to the branch with `git checkout ft-view-a-vendor-164209895`
- Activate the virtual environment and start the app with `pipenv shell && python manage.py runserver`
- Make a GET request to the route `/vendor/<vendor_id>`

`GET /vendor/<vendor_id>`

RESPONSE
`200 OK`

Response data should have the following structure
```
{
    "message" : "Vendor fetched successfully.",
    "status" : "success",
    "data":
      {
          "id": "-LZZwB7ZfrjTy4BsVM3Y",
          "name": "OgoSon",
          "location": "Mainland area",
          "description": "<description>",
          "owner": "<owner id>",
          "logo_url": "https://www.planwallpe9be975220ff4.jpg",
          "email": "<business email>",
          "phone": "<phone>"
      }
}
```

**NOTE** 
Must be a verified account.
Business account users can only view the vendors created by them.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#164209895](https://www.pivotaltracker.com/story/show/164209895)

#### Screenshots (if appropriate)
- N/A

#### Questions:
 - N/A
